AbstractFactory.Andrew.University/AbstractFactory.Andrew.University/src/College.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AbstractFactory.Andrew.University.src
+{
+    abstract class CollegeFactory
+    {
+        public abstract Course CreateCourse();
+        public abstract Student CreateStudent();
+    }
+
+    abstract class Course
+    {
+        protected List<Student> ClassList;
+        public string ClassName;
+        public abstract void AddStudent(Student x);
```

Comment:
readability:  Rename the `AddStudent` function's `Student` parameter.  Single-letter identifiers do not convey enough information.  This will cascade to concrete `Course` implementations.